makefile

```diff
@@ -19,12 +19,12 @@ format:
 
 lint:
 	python -m flake8 $(PKG)
+	python -m mypy -p $(PKG)
 	python -m ufmt check $(PKG)
 
-test:
+test: lint
```

Comment:
@amyreese , the rationale for this is not running tests on the code that foes not pass quality gates. The checks from `mypy` is a sort of linting stage. If the idea is grasped right, the test rule becomes depending on the lint rule that should be run before.

Let's discuss this addendum.

Reply:
I very much dislike this, because during development of new features, it's more important to know if tests are passing than whether or not all of the lints have been taken care of. Not being able to validate functionality of code because there's extra import or unused value is a really bad UX.